packages/rollup-plugin-html/src/RollupPluginHTMLOptions.ts

```diff
@@ -10,6 +10,13 @@ export interface InputHTMLOptions {
   path?: string;
 }
 
+export interface TagAndAttribute {
+  /** The html element tag name */
+  tagName: string;
+  /** The html attributes */
```

Comment:
Oops. The comment should say in singular form, "attribute" not "attributes". (Note: I had initially tried for a list of attributes, but would have been an extensive rewrite of this plug-in.)